<?xml version="1.0"?>
<launch>

  <!-- NOTE : This is an example launch file, please do not modify! -->

  <!--node pkg="mbot_head_pan_controller" type="mbot_head_pan_controller" name="mbot_head_pan_node" output="screen"/-->
<node pkg="mbot_head_controller_gazebo" type="mbot_head_pan_controller" name="mbot_head_pan_node" output="screen"/>


</launch>
